db_test.go

```diff
@@ -425,6 +426,52 @@ func (s *SurrealDBTestSuite) TestSmartMarshalQuery() {
 	})
 }
 
+func (s *SurrealDBTestSuite) TestConcurrent() {
+	var wg sync.WaitGroup
+	totalcoroutine := 100
+
+	user := testUser{
+		Username: "electwix",
+		Password: "1234",
+	}
+
+	s.Run(fmt.Sprintf("%d Request Concurrent select n-exists", totalcoroutine), func() {
```

Comment:
I think it would be better to have an explicit naming for the test, an example would be "ConcurrentSelectNoRowError_%dRequests" as it gives more clarity on what is happening

Reply:
Or just something along those lines to make it explicit

Reply:
done